useEffect/App.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState, useEffect } from 'react';
+import './styles.css';
```

Comment:
사소하지만 React는 import에서 제거하고, 개행 넣어주기!

```suggestion
import { useState, useEffect } from 'react';

import './styles.css';
```